### ISSUE TITLE
Postgres jsonb contains breaks logging

### ISSUE THREAD

**Issue by scirner22** [JetBrains/Exposed#682]
I implemented a simple jsonb contains key class like such.

```
class JsonbContainsKey(val expr1: Expression<*>, val expr2: JsonbExpression): Op<Boolean>() {
    override fun toSQL(queryBuilder: QueryBuilder): String {
        return "${expr1.toSQL(queryBuilder)} ?? ${expr2.toSQL(queryBuilder)}"
    }
}
```

This queries as expected, but if logging is enabled, it throws the following exception.

```
java.util.NoSuchElementException: null
        at java.base/java.util.ArrayList.next(ArrayList.java:999)
        at org.jetbrains.exposed.sql.statements.StatementKt.expandArgs(Statement.kt:90)
        at org.jetbrains.exposed.sql.Slf4jSqlDebugLogger.log(SQLLog.kt:32)
        at org.jetbrains.exposed.sql.CompositeSqlLogger.log(SQLLog.kt:50)
        at org.jetbrains.exposed.sql.CompositeSqlLogger.afterExecution(SQLLog.kt:56)
        at org.jetbrains.exposed.sql.statements.Statement.executeIn(Statement.kt:66)
        at org.jetbrains.exposed.sql.Transaction.exec(Transaction.kt:128)
        at org.jetbrains.exposed.sql.Query.count(Query.kt:344)
        at org.jetbrains.exposed.sql.IterableExKt.count(IterableEx.kt:120)
```

**Comment by qoomon** [comment]
https://github.com/JetBrains/Exposed/pull/891 should be the fix

**Comment by bog-walk** [comment]
If the problem persists in current version 0.41.1, in spite of merged [PR #891](https://github.com/JetBrains/Exposed/pull/891), please consider reopening this issue on [YouTrack](https://youtrack.jetbrains.com/newIssue?project=EXPOSED&draftId=25-4442763) and providing more details about the custom class arguments so we can assist further.
